src/main/java/uk/gov/hmcts/dts/fact/controllers/CourtsController.java

```diff
@@ -67,4 +67,10 @@ public ResponseEntity<Court> findCourtByName(@PathVariable String slug) {
     public ResponseEntity<List<CourtReference>> getCourtsBySearch(@RequestParam @Size(min = 1, max = 1) @NotBlank String prefix) {
         return ok(courtService.getCourtsByPrefixAndActiveSearch(prefix));
     }
+
+    @GetMapping(path = "/court-types/{courtTypes}")
+    @ApiOperation("Find courts by court types")
```

Comment:
We need to add an ApiDescription here to mention what possible inputs are allowed, or where to find them, and also what response types there may be